src/framework/mlt.vers

```diff
@@ -681,3 +681,10 @@ MLT_7.18.0 {
   global:
     mlt_audio_free_data;
 } MLT_7.16.0;
+
+MLT_7.20.0 {
```

Comment:
Version 7.20.0 was just release. Please change this to `MLT_7.22.0` but it is still based on `MLT_7.18.0` below since 7.20.0 has no new symbols.

Reply:
Done